### PR TITLE
Add DNS TXT and CAA record for staging-cd crossfeed

### PR DESCRIPTION
**Pull request by cduhn17**
Add DNS TXT and CAA record for staging-cd

#  #

## 🗣 Description ##

We need to update the TXT and CAA record for the DNS to create a certificate for Let'sEncrypt. terraform-docs has been run.

Add/Update the crossfeed_staging_cd_TXT

## 💭 Motivation and context ##

There are no existing ACME certs

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] *All* future TODOs are captured in issues, which are referenced
      in code comments.
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
- [x] Tests have been added and/or modified to cover the changes in this PR.
- [x] All new and existing tests pass.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ zone.  This role has a trust relationship with the users account.
 | terraform | ~> 1.0 |
 | aws | ~> 4.9 |
 
-## Providers ##
+## Providers
```
Comment (jsf9k) [inline README.md]: Please fix the lack of closing octothorps throughout this file.

**Review by jsf9k — CHANGES_REQUESTED** [review]
In the future, please give your commits more meaningful commit messages than "Update blah.tf".  The commit message should explain what is being changed and why.

File: README.md (hunk 1 above)
Comment (cduhn17) [reply README.md]: Done

**Review by jsf9k — APPROVED** [review]

**Comment by cduhn17** [comment]
@jsf9k , 
I had a failure and updated the LE value

**Comment by dav3r** [comment]
@cduhn17 I applied the latest Terraform here, which updated the value in the TXT record.  Please let us know if you're all set now.

**Comment by cduhn17** [comment]
> @cduhn17 I applied the latest Terraform here, which updated the value in the TXT record. Please let us know if you're all set now.

Can you apply again. Didnt think I was going to have to have an additional CAA for staging-cd. Thanks

**Comment by dav3r** [comment]
> > @cduhn17 I applied the latest Terraform here, which updated the value in the TXT record. Please let us know if you're all set now.
> 
> Can you apply again. Didnt think I was going to have to have an additional CAA for staging-cd. Thanks

OK, I just reapplied after the changes in https://github.com/cisagov/cool-dns-cyber.dhs.gov/pull/104/commits/b6e571c413b1612820f1619076b5569ac828c623.

**Review by dav3r — APPROVED** [review]

**Comment by cduhn17** [comment]
The DNS record update was successful

**Comment by dav3r** [comment]
@cduhn17 Can you please revert https://github.com/cisagov/cool-dns-cyber.dhs.gov/pull/104/commits/9e9c25a13038e1116eb97e3d085c46db0d30f23d so that I can merge this PR?  After that, you can recreate a new PR  with the same contents as https://github.com/cisagov/cool-dns-cyber.dhs.gov/pull/104/commits/9e9c25a13038e1116eb97e3d085c46db0d30f23d (or reopen #105) and we can get that one reviewed, applied, and merged.

We want to ensure that our Terraform code is aligned with what is actually deployed in AWS.  Thanks!

**Comment by cduhn17** [comment]
@dav3r revert is complete

**Pull request merged**